packages/@okta/vuepress-site/docs/guides/keep-me-signed-in/main/index.md

```diff
@@ -0,0 +1,277 @@
+* Set the value of `ruleId` to that of the default rule of your global session policy.
+* Set the value of `type` to `SIGN_ON`.
+* In the `signon` object, set the following values:
+  * `access`: `ALLOW`
+  * `requireFactor`: `false`
+  * `primaryFactor`: `PASSWORD_IDP_ANY_FACTOR`
+* In the `session` object, set the following:
+  * `maxSessionLifetimeMinutes`: `0`
+  * `maxSessionIdleMinutes`: `120`
+  * `userPersistentCookie`: `false`
+
+```bash
+"actions": {
+  "signon": {
+    "access": "ALLOW",
+    "requireFactor": false,
+    "primaryFactor": "PASSWORD_IDP_ANY_FACTOR",
+    "session": {
+      "usePersistentCookie": false,
+      "maxSessionIdleMinutes": 120,
+      "maxSessionLifetimeMinutes": 0
+    }
+  }
+},
+```
+
+### Update an authentication policy for post-authentication KMSI
+
+In a PUT request to the `/api/v1/policies/{policyId}/rules/{ruleId}` endpoint, include the following:
+
+* In the `appSignOn` object, set `access` to `ALLOW`.
+  * In the `keepMeSignedIn` object, set the following:
+    * `postAuth` to `ALLOWED`
+    * `postAuthPromptFrequency` to `PT168H` (Java ChronoUnit `enum` for 7 days).
+  * In the `verificationMethod` object, set the following:
+    * Set `type` to `ASSURANCE`.
+    * Set the `constraints` object to those that your org requires. See [constraints](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/#tag/Policy/operation/replacePolicyRule!path=0/actions/appSignOn/verificationMethod/0/constraints&t=request).
+    * Set `factorMode` to `2FA`.
+    * Set `reauthenticateIn` to `PT168H` (Java ChronoUnit `enum` for 7 days).
+
+```bash
+"actions": {
+  "appSignOn": {
+      "access": "ALLOW"
+    },
+      "keepMeSignedIn": {
+        "postAuth": "ALLOWED",
+        "postAuthPromptFrequency": "PT168H"
+    },
+      "verificationMethod": {
+            "contraints": [],
+            "factorMode": "2FA",
+            "reauthenticateIn": "PT168H",
+            "type": "ASSURANCE"
+    },
+  }
+```
+
+See [Replace a policy rule](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/#tag/Policy/operation/replacePolicyRule).
+
+## Customize post-authentication sign-in prompts
+
+You can customize the post-authentication prompt text, including the following:
+
+* Title
+* Subtitle
+* Accept button
+* Reject button
+
+In a PUT request to the `/api/v1/brands/{brandId}/pages/sign-in/customized` endpoint, include the following in a `postAuthKeepMeSignedInPrompt` object:
+
+* `title`: `Stay signed in?`
+* `subtitle`: `Make your selection below`
+* `acceptButtonText`: `Accept`
+* `rejectButtonText`: `Reject`
+
+```bash
+"postAuthKeepMeSignedInPrompt": {
+  "title": "Stay signed in, my friend?",
+  "subtitle": "Make your selection below",
+  "acceptButtonText": "Test accept",
+  "rejectButtonText": "Test reject"
+}
+```
+
+## Reset KMSI in your org
+
+You can reset KMSI for an individual user and clear all of their sessions.
```

Comment:
nit: We can also mention the end-user settings method to clear all sessions here as well?